MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include LICENSE README.md
-recursive-include tests *.py
\ No newline at end of file
+include LICENSE README.md CONTRIBUTING.md HISTORY.md CODE_OF_CONDUCT.md
```

Comment:
this are the files that get added to your tar.gz when you do `python setup.py sdist` we probably want to include a license there, and the readme is because setup.py expect a readme. do the rest of the files serve a deployment purpose?

Reply:
It's stuff I would expect to find under `/usr/share/doc/pystemd` if this were installed as an rpm/deb (especially HISTORY), so IMO it does make sense to have them in the tarball, given that it's just a few extra kb.